oap-server/analyzer/agent-analyzer/src/main/java/org/apache/skywalking/oap/server/analyzer/provider/trace/parser/listener/MultiScopesAnalysisListener.java

```diff
@@ -310,6 +312,7 @@ public void build() {
         slowDatabaseAccesses.forEach(sourceReceiver::receive);
```

Comment:
I think you miss this one.

Reply:
> I think you miss this one.

`slowDatabaseAccesses` is list of Metrics not SourceBuilders.

Reply:
It should be. This is another bug. DB service has over length risk too.

Reply:
> It should be. This is another bug. DB service has over length risk too.
So what fields have such risk, `databaseServiceId` or `statement`?

Reply:
I think should be service id. We need to move service name and type inside this, then add the prepare method.